Folder/main.py

```diff
@@ -0,0 +1,61 @@
+import requests
+import asset_data
+import email_notifier
+
+data_manager = asset_data.DataManager()
+data_manager.update_price_data()
+crypto_data = data_manager.crypto_data
+global_data = data_manager.global_data
+email_manager = email_notifier.EmailNotifier()
+users_data = email_manager.get_emails_data()
+
+
+# TODO Add comparisons to stock market data / top 10 companies mcap
+total_crypto_mcap = '${:,.2f}'.format(global_data["data"]["total_market_cap"]["usd"])
```

Comment:
you could check if a value exists in the dictionary before accessing like
```
// maybe this could be cleaner
if global[data]["data"] and global[data]["data"]["total_market_cap"] and global_data["data"]["total_market_cap"]["usd"])
```
accessing a key that doesn't exist in the dictionary when you expect it to is bad

Reply:
What are the circumstances that that would happen? Since I'm calling from the API, shouldn't it return data consistently unless the API itself goes through an update or like the path changes? I know that's not like a non-zero possibility

Reply:
their API can change, they can release a bug, internet issues or something unexpected. Defensive programs help your system not crash and burn when things go wrong